### PR TITLE
(brave) stop all executables being packed

### PR DESCRIPTION
**Pull request by mikeee**
The Brave package is intermittently not updated due to the large files being packed and the attempts to push these packages to the repo failing.

## Description
The feed [https://community.chocolatey.org/packages/brave/] has not been updated in its entirety for some time, the last release pushed was from mid-september.

## Motivation and Context
The issue is highlighted here: https://community.chocolatey.org/packages/brave/#comment-6036300790
The packaging script currently packs both the x86 and x64 versions together and this pushes it 200MB<

This change implements a modified install script, further inclusion of a silent uninstall script and a partial re-write of the updater script.

## How Has This Been Tested?
Tested on a W11 environment - updating followed by installing and uninstalling the package ensuring it is silent.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by RedBaron2** [comment]
@mikeee 
Might I ask why keep the `legal/verification.txt` file? This is only used for embedded packages. Packages that are downloaded at run time.  Do not need this file.  The download is checked with the supplied checksum in the package.

**Comment by mikeee** [comment]
> @mikeee Might I ask why keep the `legal/verification.txt` file? This is only used for embedded packages. Packages that are downloaded at run time. Do not need this file. The download is checked with the supplied checksum in the package.

I'm not too sure about the specific conventions of chocolatey packages being validated/verified. I kept the breaking changes to a minimum in case anyone was actively using the verification.txt file for any reason. If this is something strongly opposed I can remove it 👍

File: automatic/brave/update.ps1 [hunk 1]
```diff
@@ -1,97 +1,89 @@
...
-  catch {
-    $streams.stable = 'ignore'
-    Write-Host "No stable release is available"
+  beta = @{
+    Title           = 'Brave Browser (Beta)'
+    ReleaseUrl      = 'https://brave-browser-downloads.s3.brave.com/latest/beta.version'
+    DownloadUrl32   = 'https://github.com/brave/brave-browser/releases/download/v{0}/BraveBrowserStandaloneSilentBetaSetup32.exe'
+    DownloadUrl64   = 'https://github.com/brave/brave-browser/releases/download/v{0}/BraveBrowserStandaloneSilentBetaSetup.exe'
+    VersionFormat   = '{0}-beta'
+    IconUrl         = 'https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@a23ca306537e2537a574ddc55e9c28dc1151ab30/icons/brave-beta.svg'
```
Comment (AdmiringWorm) [inline automatic/brave/update.ps1:18]: THese should still be located inside of the `au_GetLatest` function, not outside of it.

Please don't reorder the file, or change the logic if it isn't needed.

File: automatic/brave/tools/chocolateyInstall.ps1 [hunk 2]
```diff
@@ -3,8 +3,12 @@
...
 $packageArgs = @{
   packageName = $env:ChocolateyPackageName
-  file        = "$toolsPath\BraveBrowserStandaloneSilentSetup32.exe"
-  file64      = "$toolsPath\BraveBrowserStandaloneSilentSetup.exe"
+  url                    = ''
+  url64bit               = ''
+  checksum               = ''
+  checksum64             = ''
+  checksumType           = 'sha256'
+  checksumType64         = 'sha256'
```
Comment (AdmiringWorm) [inline automatic/brave/tools/chocolateyInstall.ps1:11]: Not sure if I agree with moving this to a completely remote package (a package that only downloads during installation).

If the 64bit installer isn't too large, the 64bit installer should still be embedded but download the 32bit if requested.

Can you please make the necessary changes to make that happen?

File: automatic/brave/tools/chocolateyUninstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿$registry = Get-UninstallRegistryKey -SoftwareName "brave*"
```
Comment (AdmiringWorm) [inline automatic/brave/tools/chocolateyUninstall.ps1:1]: I thought the automatic uninstaller would be able to uninstall the software.
Has this changed and is no longer picked up automatically?

This uninstall script is also very error prone, and will fail the uninstallation if the program had already been uninstalled.

See https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/7zip.install/tools/chocolateyUninstall.ps1 as an example on how the uninstall script should be implemented.

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
A few things need to be changed before we can accept this PR.

**Pull request closed without merging**